.travis.yml

```diff
@@ -1,8 +1,18 @@
 language: java
 
+os: linux
+
 # Need to select an older Ubuntu distribution that supports JDK7
 dist: trusty
 
+jdk:
+  - openjdk7
+  - oraclejdk8
+  - openjdk8
+  - oraclejdk9
+  - openjdk11
+  - openjdk14
```

Comment:
1) is early access of openjdk15 available yet?
2) is oraclejdk11 avaliable?

Reply:
I didnt try these versions but Travis has an option for allowing specific builds to fail without fail the complete build. Some projects adds the latest or beta versions in this list.

If you want i can create another PR to add these and other versions :dancers: